### PR TITLE
Feature/rid 002 dashboard page

### PR DESCRIPTION
**Pull request by georgebullock**

### CODE REVIEW

File: src/routes/dashboard/dashboard.module.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	z-index: 10;
+
+	@include media('>bp-tablet-m') {
+		position: fixed;
+		left: 0;
+		height: 100vh;
+		width: (1/12) * 100%;
+	}
+
+	@include media('>bp-tablet-m') {
```
Comment (Sebastian-Fitzner) [inline src/routes/dashboard/dashboard.module.scss:35]: Duplicate

File: src/shared/Grid/Grid.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	alignItems,
+	justify,
+	lg,
+	md,
+	sm,
+	children
+}: GridProps) => {
+	const isRow = row || column;
+
+	const classes: string =
```
Comment (Sebastian-Fitzner) [inline src/shared/Grid/Grid.tsx:45]: I would try to use `classnames` here.

**Review by Sebastian-Fitzner — APPROVED** [review]
Looks all good!

File: src/routes/dashboard/dashboard.module.scss (hunk 1 above)
Comment (georgebullock) [reply src/routes/dashboard/dashboard.module.scss:35]: Fixed. Thanks!

File: src/shared/Grid/Grid.tsx (hunk 2 above)
Comment (georgebullock) [reply src/shared/Grid/Grid.tsx]: Do you mean `classnames` as the variable name or "classnames" the npm package?

Comment (Sebastian-Fitzner) [reply src/shared/Grid/Grid.tsx]: after taking a look, i would consider this one: https://github.com/lukeed/clsx
`classnames` package is not really maintained anymore...

**Pull request merged**